test/mosaic-integration-tests/src/test/java/org/eclipse/mosaic/test/SumoPredefinedScenarioIntegrationIT.java

```diff
@@ -66,4 +68,15 @@ public void applicationAmbassadorReceivesAllRoutes() throws Exception {
         LogAssert.contains(simulationRule, VEH_1_SUMO, ".*4 routes are known to the SimulationKernel.*");
     }
 
+    @Test
+    public void sumoVehicleFirstUpdateAtStartup() throws Exception {
+        for (String sumoVehicleLog : Lists.newArrayList(VEH_1_SUMO, VEH_2_SUMO)) {
+            String onStartupTime = LogAssert.getMatches(simulationRule, sumoVehicleLog, 1,
+                    ".*Startup @(.*?) s: I'm a vehicle defined in SUMO route file.*").get(0);
+            String firstUpdateTime = LogAssert.getMatches(simulationRule, sumoVehicleLog, 1,
+                    ".*First update @(.*?) s .*").get(0);
```

Comment:
yet you only assert the first match :D but I see that this extension could be useful for other tests ;)

Reply:
Yes. I feel like this feature has use cases in the future. For my case I just want to capture two separate values from two separate regular expressions.